git/git.go

```diff
@@ -490,13 +490,45 @@ func (c *gitConfig) IsGitVersionAtLeast(ver string) bool {
 	return IsVersionAtLeast(gitver, ver)
 }
 
+// Prepend Git config instructions to disable Git LFS filter
+func PrependConfigToDisableLFS(args ...string) []string {
```

Comment:
I am in favor of the wrapper functions, with two recommendations:

1. Un-export `Exec()` and `SimpleExec()`.
1. Make `exec()` and `simpleExec()` execute their arguments literally, without implicitly prepending the LFS-disabling arguments to Git. If a caller wants to run a Git command that doesn't then go back and run LFS, they should call `exec` like: `exec(disableLfs(args...))`.